planemo/commands/cmd_autoupdate.py

```diff
@@ -0,0 +1,64 @@
+"""Module describing the planemo ``autoupdate`` command."""
+import click
+
+from planemo.exit_codes import (
+    EXIT_CODE_GENERIC_FAILURE,
+    EXIT_CODE_OK
+)
+from planemo.io import (
+    coalesce_return_codes,
+    info
+)
+from planemo.tools import (
+    is_tool_load_error,
+    yield_tool_sources_on_paths
+)
+
+from planemo import options, autoupdate
+from planemo.cli import command_function
+from planemo.config import planemo_option
+
+
+def dry_run_option():
```

Comment:
not sure if this should be moved to `options.py` or not

Reply:
It is totally fine here, especially if only one command is using it.